src/utils.js

```diff
@@ -2,14 +2,26 @@ import React, { isValidElement, cloneElement } from 'react';
 
 export const matcher = ({ option, searchTerm = '', searchIndices }) => {
   searchTerm = searchTerm.trim().toLowerCase();
-  if (typeof option === 'string') {
-    return option.toLowerCase().indexOf(searchTerm) !== -1;
-  }
+
   if (searchIndices) {
     return makeArray(searchIndices).some(index => {
-      return (option[index] || '').toLowerCase().indexOf(searchTerm) !== -1;
+      let value = option[index];
+      return (
+        !isNone(value) &&
```

Comment:
`isNone` func is missing

Reply:
Ah, totally my mistake. When I brought it over to the branch I must have forgotten to copy it.